wgpu/src/lib.rs

```diff
@@ -3866,6 +3876,10 @@ impl CommandEncoder {
     /// Begins recording of a render pass.
     ///
     /// This function returns a [`RenderPass`] object which records a single render pass.
+    ///
+    /// As long as the returned  [`RenderPass`] has not ended,
+    /// any recording operation on this command encoder causes an error and invalidates it.
+    /// (Note that the lifetime constraint protects against this.)
```

Comment:
Out of context, this sounds weird to the moderately experienced Rust programmer: “You must not do this, but also, you can’t do this.” I think it should contain the same note as below. Suggested wording (not quite a copy of below but tweaked):

```suggestion
    /// (Note that the `'pass` lifetime relationship normally protects against this,
    /// but can be overridden by [`ComputePass::make_static()`].)
```